### PR TITLE
Create 0011-researcher-in-residence.md

### PR DESCRIPTION
**Pull request by LucasMHrug**
# Description

Please include a summary of the changes and the related issue.

### CODE REVIEW

File: docs/ADRs/0011-researcher-in-residence.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- The are independent as they are employed and financed by their respective university.
+- The results of their research, and thus processed data, are published in an academic journal.
+
+The following conditions apply to the researcher in residence.
+
+- The research is able to access and analyze documents relevant to the research (ex. Notes/minutes taken). These
+  documents are submitted to a member of the team for review prior to being processed.
+- Any data collected and analyzed via interviews is done on the basis of informed consent.
+- The data collected is not shared with other parties and is handled ethically by the researcher.
+- No information, aggregation, or summary of information from the communication channel of the team (mattermost) is
```
Comment (anneschuth) [inline docs/ADRs/0011-researcher-in-residence.md:39]:
```suggestion
- No information, aggregation, or summary of information from the communication channel of the team (Mattermost) is
```

File: docs/ADRs/0011-researcher-in-residence.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## Decision
+
+We have decided to include a researcher in residence as a member of our team.
+
+The researcher in residence takes the following form:
+
+- They join and participate in refinement meetings and other meetings of relevance.
+- They share their reflections and present their (interim) research findings.
+- They are provided access to the communication channel of the team (mattermost).
```
Comment (anneschuth) [inline docs/ADRs/0011-researcher-in-residence.md:28]:
```suggestion
- They are provided access to the communication channel of the team (Mattermost).
```

File: docs/ADRs/0011-researcher-in-residence.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+In light of aforementioned form and conditions above, we see no foreseeable significant risks.
+
+## Consequences
+
+Including a researcher in residence makes it easier for them to conduct research within both the team and the wider
+organization where the AI validation team operates. This benefits the quality of the research findings and the feedback
+provided to the team and organization.
+
+## More Information
```
Comment (anneschuth) [inline docs/ADRs/0011-researcher-in-residence.md:52]:
```suggestion
```

File: docs/ADRs/0011-researcher-in-residence.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+- The research is able to access and analyze documents relevant to the research (ex. Notes/minutes taken). These
+  documents are submitted to a member of the team for review prior to being processed.
+- Any data collected and analyzed via interviews is done on the basis of informed consent.
+- The data collected is not shared with other parties and is handled ethically by the researcher.
+- No information, aggregation, or summary of information from the communication channel of the team (mattermost) is
+  collected, processed, or analyzed by the researcher.
+
+## Risks
+
+In light of aforementioned form and conditions above, we see no foreseeable significant risks.
```
Comment (anneschuth) [inline docs/ADRs/0011-researcher-in-residence.md:44]:
```suggestion
Risks around a potential chilling effect (team members not feeling free to express themselves) are mitigated by the conditions we impose. 
In light of aforementioned form and conditions above, we see no further significant risks. 
```

**Review by anneschuth — DISMISSED** [review]
Congrats on your first PR! 🎉

**Comment by berrydenhartog** [comment]
shall we squash this PR ?

**Review by anneschuth — APPROVED** [review]

**Pull request merged**